From 963b0d11d6f25ec8a8bd95e9fcea2d9933e31bcf Mon Sep 17 00:00:00 2001
From: Adrian Sampson <adrian@radbox.org>
Date: Sat, 6 Apr 2013 16:35:17 -0700
Subject: [PATCH 1/3] fix VFS tests

The configuration was not loaded for these tests because they didn't inherit
the common test harness. These failures were hidden on my system because of
some kind of dependency on another test.
---
 test/test_vfs.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/test/test_vfs.py b/test/test_vfs.py
index 04ff3d6..755311b 100644
--- a/test/test_vfs.py
+++ b/test/test_vfs.py
@@ -18,8 +18,9 @@ from _common import unittest
 from beets import library
 from beets import vfs
 
-class VFSTest(unittest.TestCase):
+class VFSTest(_common.TestCase):
     def setUp(self):
+        super(VFSTest, self).setUp()
         self.lib = library.Library(':memory:', path_formats=[
             ('default', 'albums/$album/$title'),
             ('singleton:true', 'tracks/$artist/$title'),
-- 
1.8.1.5

